content/blog/greenlake-platform-infrastructure-monitoring-with-apache-skywalking.md

```diff
@@ -0,0 +1,144 @@
+---
+title: GreenLake Platform Infrastructure monitoring with Apache SkyWalking
+date: 2022-12-13T08:52:15.545Z
+author: Thirukkannan M
+authorimage: /img/thirupp_192x192.jpg
+thumbnailimage: /img/thirupp_192x192.jpg
+disable: false
+tags:
+  - Infrastructure Monitoring
+  - Apache SkyWalking
+---
+**Overview**
+
+HPE GreenLake Central is an advanced software-as-a-service platform that provides you with a consistent cloud experience for all your applications and data - on-premises or off-premises. It provides you with insights and controls to manage your hybrid IT estate, complementing your use of public clouds and data centers. HPE GreenLake Central gives you the ability to choose where and how to place your workloads and data, and - through the services you purchase - enables you to monitor security, capacity, resource utilization, and costs.
```

Comment:
Consider rewriting lines 14-19 with something like this:

SkyWalking is a popular open-source application performance monitoring (APM) tool used to collect, analyze, aggregate, and visualize data from services and cloud-native infrastructures. It provides an easy way to maintain a clear view of your distributed systems, even across clouds. You can use it with HPE GreenLake for Private Cloud Enterprise, the managed, pay-per-use Private Cloud as a Service (PCaaS) solution to enhance monitoring and observability of your workloads.

Because HPE GreenLake for Private Cloud Enterprise provides you with a flexible, self-service environment where you can implement a hybrid approach and run different kinds of bare metal, virtual machine, and containerized workloads, you can take advantage of additional apps and tools like SkyWalking to customize your situation and meet your specific needs.

In this post, I will focus on how you can use SkyWalking and Open Telemetry to do infrastructure host monitoring while taking advantage of how HPE GreenLake provides you with a consistent cloud experience across all your applications and data, whether on- or off-premises. (Now, if you're going to show some additional integration of the two, you might want to add some of that in here, too. IF the HPE GreenLake Central piece about "enables you to monitor security, capacity, resource utilization, and costs" is covered in this post, you can add those pieces in, too, but add them in context.